baremetal/metal3machine_manager_test.go

```diff
@@ -187,21 +187,21 @@ func bmhSpecNoImg() *bmh.BareMetalHostSpec {
 func m3mObjectMetaWithValidAnnotations() *metav1.ObjectMeta {
 	return &metav1.ObjectMeta{
 		Name:            "mym3machine",
-		Namespace:       "myns",
+		Namespace:       namespaceName,
 		OwnerReferences: []metav1.OwnerReference{},
 		Labels: map[string]string{
 			clusterv1.ClusterLabelName: clusterName,
 		},
 		Annotations: map[string]string{
-			HostAnnotation: "myns/myhost",
+			HostAnnotation: fmt.Sprintf("%s/myhost", namespaceName),
```

Comment:
In these situations I would just add the strings together since it is so simple.
```suggestion
			HostAnnotation: namespaceName + "/myhost"
```
It probably doesn't matter much, but worth considering.

Reply:
+1

Reply:
Thanks. Changed.